Library/Vertex.cs

```diff
@@ -23,6 +23,12 @@ namespace BMeshLib
      * The vertex position does not affect topological algorithms, but is used by
      * commodity functions that help finding the center of an edge or a face.
      */
+    /// <summary>
+    /// Corresponds to a point in space, and is used by <see cref="Edge"/> and <see cref="Face"/> to construct a <see cref="BMesh"/>. 
+    /// </summary>
+    /// <remarks>
+    ///  Multiple <see cref="Edge"/>s and <see cref="Face"/>s can use the same <see cref="Vertex"/>, and multiple vertices can be located at the exact same position.
+    /// </remarks>
```

Comment:
Add the other remark about data structure details.

Reply:
And what I said about the Loop comment goes for here too. I feel like some of the info in the technical comment could be added, but I don't think that it should all be added.

Reply:
So same answer ^^ Put it in a remark or even in an invented `<dev>` téag but still include the whole of it because I need it to remember.